src/types/config.types.ts

```diff
@@ -7,7 +7,7 @@ export interface GlobalInfinityQuery {
   id: string;
   query: InfinityQuery;
 }
-export type AuthType = 'none' | 'basicAuth' | 'apiKey' | 'bearerToken' | 'oauthPassThru' | 'digestAuth' | 'aws' | 'azureBlob' | 'oauth2';
+export type AuthType = 'none' | 'basicAuth' | 'apiKey' | 'bearerToken' | 'oauthPassThru' | 'digestAuth' | 'aws' | 'azureBlob' | 'oauth2' | 'microsoft';
```

Comment:
Azure is misleading here, because we are authenticating against Microsoft Entra ID. And the token can be used for non-azure service as well. For example Microsoft Graph API and other Microsoft 365 related services.